### PR TITLE
Add iOS target

### PR DESCRIPTION
**Pull request by matthewdias**

### CODE REVIEW

**Comment by DattatreyaReddy** [comment]
Does it matter?

**Comment by matthewdias** [comment]
you can use it with AltStore

**Comment by matthewdias** [comment]
Successfully builds and installs on device through AltStore image

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -50,8 +53,9 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: "12.x"
+        
       - name: Enable desktop support
-        if: matrix.target != 'android'
+        if: matrix.target != 'android' && matrix.target != 'ios' && matrix.target != 'web'
```
Comment (matthewdias) [inline .github/workflows/publish.yml:58]: this was configuring desktop support for more targets than was necessary

**Pull request merged**